src/client/systems/InputController.hpp

```diff
@@ -22,6 +28,8 @@ class InputController {
 
     InputDef::ReliableInput m_reliableInput;
     InputDef::UnreliableInput m_unreliableInput;
+
+    std::unique_ptr<GreedyBot> m_bot;
```

Comment:
This doesn't need to be a unique_ptr, it can just be allocated on the stack considering you're not passing it around or transferring ownership.
`GreedyBot m_bot`

Reply:
nice didn't know that, just copy pasta'd input controller